lib/screen/home/selected_workout.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:event_bloc/event_bloc.dart';
+import 'package:flutter/material.dart';
+
+class SelectedWorkout extends StatelessWidget {
```

Comment:
Usually I prefer adding 'Screen' or 'Widget' as a suffix to my Widgets. It's a bit unclear what layer this class belongs to (this case it could be mistaken as a BLoC layer class)